### PR TITLE
Remove legacy Admin UI for core field types

### PR DESCRIPTION
**Pull request by timleslie**
See #5025.

Third in the series to remove legacy field views.

### CODE REVIEW

**Review by JedWatson — APPROVED** [review]

File: design-system/packages/core/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.13.10",
+    "@emotion/cache": "^10.0.27",
```
Comment (mitchellhamilton) [inline design-system/packages/core/package.json:19]: Why did you add this?

Comment (timleslie) [reply design-system/packages/core/package.json:19]: I was getting CI failures in the `document` field and the `docs-next` page. It seems like these was an implicit dependency being provided by one of the `@arch-ui` packages, which I traced back this far.

Not sure if this is the right solution, but so far it seems to at least make our tests pass 🤷‍♂️ .

**Review by mitchellhamilton — APPROVED** [review]
